### PR TITLE
chore(test): update e2e test to leverage typescript

### PR DESCRIPTION
**Pull request by dtfiedler**
This should help sniff out type issues before releases.

### CODE REVIEW

File: tests/e2e/esm/index.test.ts [hunk 1]
```diff
@@ -382,7 +366,7 @@ describe('e2e esm tests', async () => {
 
     it('should be able to get balances of a specific to first page', async () => {
       const balances = await ario.getBalances({
-        cursor: 1000000,
+        cursor: '1000000',
```
Comment (atticusofsparta) [inline tests/e2e/esm/index.test.ts:369]: im a little confused by this test - this cursor seems configured to balances nots addresses?

**Review by atticusofsparta — APPROVED** [review]

File: tests/e2e/esm/index.test.ts (hunk 1 above)
Comment (dtfiedler) [reply tests/e2e/esm/index.test.ts:369]: Balances are a valid cursor, but the type type expects a string.

**Pull request merged**